airbyte-integrations/connectors/source-google-sheets/source_google_sheets/source.py

```diff
@@ -259,6 +259,11 @@ def read(
                 ) from e
             else:
                 logger.info(f"{e.status_code}: {e.reason}. {error_description}")
+                raise AirbyteTracedException(
```

Comment:
@Reidsy, can you update the connector version in the metadata.yaml and pyproject.toml files? Additionally, please add an entry in the changelog connector docs (docs/integrations/source/...).

Reply:
Bumped version to `0.7.4` and added information to changelog